ElectronClient/app/gui/NotebookPropertiesDialog.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { useState, useEffect } from 'react';
+
+const { _ } = require('lib/locale.js');
+const { themeStyle } = require('../theme.js');
+const DialogButtonRow = require('./DialogButtonRow.min');
+
+const Folder = require('lib/models/Folder.js');
+
+interface NotebookPropertiesDialogProps {
+	theme: number,
+	folderId: string,
+	onClose: Function,
+}
+
+export default function NotebookPropertiesDialog(props: NotebookPropertiesDialogProps) {
+	console.info('Render NotebookPropertiesDialog');
+
+	const [notebook, setNotebook] = useState<any>({});
```

Comment:
Please use the term "folder" internally. I know it's inconsistent that it's called "Notebook" in the UI and "folder" internally, but at least we can try to keep it consistent internally. So nearly all the time it should be called "folder", except in strings in the UI.